internal/tezos/new_block_listener_test.go

```diff
@@ -13,6 +14,32 @@ import (
 
 func TestNewBlockListenerOK(t *testing.T) {
 	ctx, c, mRPC, done := newTestConnector(t)
+
+	c.blockListener.blockPollingInterval = 1
+	mRPC.On("GetHeadBlock", mock.Anything).Return(
+		&rpc.Block{
+			Hash: tezos.MustParseBlockHash("BMBeYrMJpLWrqCs7UTcFaUQCeWBqsjCLejX5D8zE8m9syHqHnZg"),
+			Header: rpc.BlockHeader{
+				Predecessor: tezos.MustParseBlockHash("BLc1BjmZ7WevMoMoj8jxh4k2wLoRqoMUxjrQuDmKzAsApfRRjFL"),
+				Level:       12345,
+			},
+		}, nil).Maybe()
+
+	req := &ffcapi.NewBlockListenerRequest{
+		ID:              fftypes.NewUUID(),
+		ListenerContext: ctx,
+		BlockListener:   make(chan<- *ffcapi.BlockHashEvent),
+	}
+
+	res, _, err := c.NewBlockListener(ctx, req)
+	done()
```

Comment:
Let's use 'defer done()' instead (at line 17) to be sure, that it was closed

Reply:
Actually the idea of this fix is moving done here that don't wait additional asserts execution as it was for done for example TestBlockListenerStartGettingHighestBlockRetry. If you compare with the test below the difference is that we just slightly increased time of execution 1 sec + asserts time of execution before done() will be executed and as a result MonitorBlockHeader is executed.
As for point `to be sure, that it was closed` since this is test and its behaviour should be deterministic so we know that no errors for it then done() should be executed as well.

Reply:
See that the test was failed again so let's remove it - coverage can't be decreased. Let's monitor it with verbose option we should see more details